docs/quickstarts/twitter.rst

```diff
@@ -54,8 +54,51 @@ run:
     $ export OAUTHLIB_INSECURE_TRANSPORT=1
     $ python twitter.py
 
-Visit http://localhost:5000 in your browser, and you should start the OAuth dance
-immediately.
+Visit ``http://127.0.0.1:5000`` in your browser, and you should start the OAuth dance
+immediately after you click on the Signin with Twitter hyperlink.
```

Comment:
Which hyperlink are you referring to? This is confusing.

Reply:
Because Twitter wants 127.0.0.1 instead of localhost, I thought it should be consistent.